### ISSUE TITLE
Incorect use of translated plural form (observed in Czech translation)

### ISSUE THREAD

**Issue by p-bo** [glpi-project/glpi#3733]
How to reproduce:
Download .po export from Transifex (9.2 branch), convert it into .mo and replace file in installation of current version.
Note: Data on Transifex are in right format - no issue here, it is in representation in application itself.

What is expected / what you get now:

where is plural form for count of 2,3 and 4 items - place this form (1st plural form on Transifex)
Examples:
Nástroje > Uložená hledání
Podpora > Problémy
/
But instead of this, there are appearing forms for count of 5 and more items (2nd plural form on Transifex). So above example is malformed as:

Nástroje > Uložených hledání
Podpora > Problémů

To recap - please use in Czech (and possibly other languages?) 2nd plural form only where it belongs.

**Comment by trasher** [comment]
Problem is that GLPI does not handle the way plural are handled; this is dedicated to a third party component (Zend Translator).
As far as I can see, everything is OK on GLPI side; maybe a bug in the component? I've not been able to find any existing or fixed issue related on their side.

**Comment by trasher** [comment]
I've just received a "news" email from transifex; that is about issues o some languages (including cs_CZ); that will be deployed may, 15th.

**Comment by p-bo** [comment]
Great, hope it helps not only in this, but unset charset variable too...

Thanks for info :)

**Comment by trasher** [comment]
Hi, could you please test again and tell us if the issue has ben solved (assuming transifex has changed as announced)?
You can try on the demo website; it's been updated this morning with all latest translations ;)

**Comment by p-bo** [comment]
Right now, we translators have sorted out some misunderstanding between us about usage of these new forms on Transifex. Right now it is in intended form in translation for 9.3 branch on Transifex (9.4 is locked - why?). Could it be possible to install this into demo to see result please? Thanks :)

**Comment by trasher** [comment]
I've just retrieved and updated locales (including cs_CZ) on the demo.
9.4 is locked right now because we've asked a big work on the 9.3, and they're similar. In a few weeks, we'll take resources from 9.3, update  9.4 and then unlock it. They will differ after that.

**Comment by p-bo** [comment]
For me it looks OK, waiting for opinion of colleague translator.

**Comment by p-bo** [comment]
Colleague translator confirmed - it's solved! Many thanks :)

**Comment by trasher** [comment]
Great; thanks for the feedback :)
